pkg/metrics/init.go

```diff
@@ -129,4 +131,24 @@ func GetQueueMetrics(name string) CoreQueueMetrics {
 		m.queues[name] = queueMetrics
 		return queueMetrics
 	}
+}
+
+func FormatMetricName(metricName string) string {
```

Comment:
Can protect this function since this is supposed to be only called within the package
---> formatMetricsName

Reply:
Done